core/src/components/cat-datepicker/cat-datepicker.locale.ts

```diff
@@ -23,9 +23,11 @@ function daysForLocale(language: string, weekday: 'long' | 'short' | 'narrow' =
 }
 
 function monthsForLocale(language: string, month: 'long' | 'short' = 'long') {
-  const date = new Date(0);
-  const format = new Intl.DateTimeFormat(language, { month }).format;
-  return [...Array(12).keys()].map(month => format(new Date(date.getTime()).setUTCMonth(month)));
+  return Array.from({ length: 12 }, (_, i) => {
+    const date = new Date();
+    date.setMonth(i);
+    return date.toLocaleString(language, { month: month });
```

Comment:
A bit nitpicky, but you can write
`return date.toLocaleString(language, { month });`

Reply:
Done thank you